src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -142,27 +147,31 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
+  const { userCase, isJointApplication, language } = content;
   const aosSubmitted =
-    !content.isJointApplication &&
-    (content.userCase.applicant2StatementOfTruth ||
-      content.userCase.aosStatementOfTruth ||
-      content.userCase.documentsUploaded?.find(doc => doc.value.documentType === DocumentType.RESPONDENT_ANSWERS));
-  const hasCertificateOfService = content.userCase.alternativeServiceOutcomes?.find(
+    !isJointApplication &&
+    (userCase.applicant2StatementOfTruth ||
+      userCase.aosStatementOfTruth ||
+      userCase.documentsUploaded?.find(doc => doc.value.documentType === DocumentType.RESPONDENT_ANSWERS));
+  const hasCertificateOfService = userCase.alternativeServiceOutcomes?.find(
     alternativeServiceOutcome => alternativeServiceOutcome.value.successfulServedByBailiff === YesOrNo.YES
   );
-  const hasCertificateOfDeemedOrDispensedService = content.userCase.alternativeServiceOutcomes?.find(
+  const hasCertificateOfDeemedOrDispensedService = userCase.alternativeServiceOutcomes?.find(
     alternativeServiceOutcome =>
       alternativeServiceOutcome.value.alternativeServiceType === AlternativeServiceType.DEEMED ||
       alternativeServiceOutcome.value.alternativeServiceType === AlternativeServiceType.DISPENSED
   );
-  const hasCertificateOfEntitlement = content.userCase.coCertificateOfEntitlementDocument;
-  const hasConditionalOrderGranted = content.userCase.coConditionalOrderGrantedDocument;
-  const hasConditionalOrderAnswers = content.userCase.documentsGenerated?.find(
+  const hasCertificateOfEntitlement = userCase.coCertificateOfEntitlementDocument;
+  const hasConditionalOrderGranted = userCase.coConditionalOrderGrantedDocument;
+  const hasConditionalOrderAnswers = userCase.documentsGenerated?.find(
     doc => doc.value.documentType === DocumentType.CONDITIONAL_ORDER_ANSWERS
   );
-  const hasConditionalOrderApplication = content.userCase.documentsGenerated?.find(
+  const hasConditionalOrderApplication = userCase.documentsGenerated?.find(
     doc => doc.value.documentType === DocumentType.CONDITIONAL_ORDER_APPLICATION
   );
+  const hasFinalOrderApplication =
```

Comment:
Feel like this name might be a bit misleading - the condition is checking that we both have FINAL_ORDER_APPLICATION document and we are in FinalOrderRequested state.

Reply:
Altered to hasFinalOrderApplicationAndFinalOrderRequested